SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -214,6 +185,11 @@ void USpatialNetConnection::SetHeartbeatEventTimer()
 			}
 		},
 		GetDefault<USpatialGDKSettings>()->HeartbeatIntervalSeconds, true, 0.0f);
+
+	if (APlayerController* Controller = GetPlayerController(GetWorld()))
```

Comment:
Why is `GetPlayerController()` used here instead of the value in `PlayerController`?